src/main/java/frc/robot/maps/Valkyrie.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.maps;
+
+import org.littletonrobotics.junction.Logger;
+import org.littletonrobotics.junction.networktables.NT4Publisher;
+import org.littletonrobotics.junction.wpilog.WPILOGWriter;
+
+import com.chopshop166.chopshoplib.maps.RobotMapFor;
+import com.chopshop166.chopshoplib.motors.CSSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.PowerDistribution;
+import edu.wpi.first.wpilibj.PowerDistribution.ModuleType;
+import frc.robot.maps.subsystems.ArmRotateMap;
+import frc.robot.maps.subsystems.LedMap;
+
+@RobotMapFor("Valkyrie")
+public class Valkyrie extends RobotMap {
+
+    @Override
+    public LedMap getLedMap() {
+        return new LedMap(0, 30);
+    }
+
+    @Override
+    public ArmRotateMap getArmRotateMap() {
+        CSSparkMax motor = new CSSparkMax(18, MotorType.kBrushless);
```

Comment:
In here is where you'd configure the PID controller's tolerance for being at the setpoint